### PR TITLE
poac: install completions

### PR DESCRIPTION
**Pull request by ken-matsui**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

Related to: https://github.com/Homebrew/homebrew-core/pull/106329#issuecomment-1191737282

### CODE REVIEW

File: Formula/poac.rb [hunk 1]
```diff
@@ -49,6 +49,8 @@ def install
     system "cmake", "--install", "build"
 
     man1.install (buildpath/"src/etc/man/man1").children
```
Comment (carlocab) [inline Formula/poac.rb:51]: While we're here, let's simplify this a little
```suggestion
    man.install "src/etc/man/man1"
```

Comment (ken-matsui) [reply Formula/poac.rb:51]: Thanks!

**Review by carlocab — APPROVED** [review]
Thanks!

**Comment by ken-matsui** [comment]
Thank you for your review!

**Pull request closed without merging**